opcua/server/user_manager.py

```diff
@@ -72,6 +72,11 @@ def check_user_token(self, isession, token):
             except Exception as exp:
                 self.logger.warning("Unable to decrypt password")
                 return False
+        else:
+            try:
+                passwd = passwd.decode('utf-8')
+            except:
+                self.logger.warning("Unable to decode password")
```

Comment:
Is a warning enough? Aren't we fucked and should return am error?

Reply:
I tested this with the UAExpert and the password was a bytestring on None, too.

Reply:
If the password is not to decode, then it is to handle by ceck_user fro me. So it should put the password thru and user has to handle errors. What do you think?

Reply:
Ok. Make sense

Reply:
Perfect. The test in Python2.7 crashed, but Python3 is done. Maybe it is to fix Travis by run the 2.7 tests again.